app/views/two_factor_authentications/_confirm.haml

```diff
@@ -0,0 +1,22 @@
+.row
+  .col-md-12
+    %h3 Confirm two factor authentication
+    %p Two factor authentication is not fully activated yet
+
+    .row
+      .col-md-6
+        %p Please scan the QR code with a TOTP capable app (eg Google authenticator).
+        = raw RQRCode::QRCode.new(@qrcode_uri).as_svg(padding: 0, module_size: 5)
```

Comment:
- Rails/OutputSafety: Tagging a string as html safe may be a security risk.
- Avoid using instance variables in partials views